Formula/imapsync.rb

```diff
@@ -104,12 +114,6 @@ def install
   end
 
   test do
-    output = shell_output("#{bin}/imapsync --dry", 255)
```

Comment:
@basictheprogram is there anyway that we can improve the test part? 

Really appreciate your efforts on fixing the build issue.

Reply:
My first attempt at building a package. I just fixed the poor testing that was done in the previous build. :-( 

What are some other tests that can be performed without have a source and/or destination imap mailbox to confirm things actually get transferred?

Reply:
Do you think this would be helpful. If there is nothing can be truly improved, I am totally cool with the current test setup though.

```
imapsync --dry \
       --host1 test1.lamiral.info --user1 test1 --password1 secret1 \
       --host2 test2.lamiral.info --user2 test2 --password2 secret2
```

Thanks!

Reply:
Well need some testing of actual features though. The current version check isn't enough cause it rarely breaks on compilation issues.

Reply:
Added your testing @chenrui333